tests/all/main.rs

```diff
@@ -48,6 +48,14 @@ fn project() -> Project {
                 "webpack.config.js".to_string(),
                 r#"
                 const path = require('path');
+                const fs = require('fs');
+
+                let nodeModules = {};
+                fs.readdirSync('node_modules').filter(function(x) {
+                    return ['.bin'].indexOf(x) === -1;
```

Comment:
This seems like a roundabout way of saying

```js
return x !== '.bin';
```

am I misreading this?

Reply:
yeah 😁